### DISCUSSION TITLE
Plans for MPC menu in the GUI?

### DISCUSSION THREAD

**Discussion by SargonofAssyria** [Jyers/Marlin#2061] [Ideas]
I love the GUI, so I wondered if there are any plans to incorporate the MPC menu into this GUI?
I know it is experimental, but it is looking good.

**Comment by SargonofAssyria** [comment]
Sorry, now I see that nothing has been done on the GUI in the last 6 months. So this came to an end?

**Comment by RV-from-be** [comment]
Jyers gave up development a long time ago.  Another developer took over this development, first here, since December 2021 with encouragement from Jyers.  In July 2022, a person (L..77) came to rot this place by illicitly copying the work of this developer and declaring himself as the author.  Our developer, like many speakers here, has gone elsewhere.  You can find his latest developments on the Discord JSTECH (https://discord.gg/NS4zYd23dX).  You will find your happiness there, like MPC. I use its very latest firmware v3.6 integrating the very latest features of Marlin.  No other firmware for Ender3V2 offers them.

**Comment by SargonofAssyria** [comment]
Tnx, but I got an invalid invitation, so I got not get in. Could you verify that?

**Comment by SargonofAssyria** [comment]
Never mind the invitation worked on my phone. Tnx. again.

**Comment by tmuehlhoff** [comment]
The Discord is in French..... Any way to join in in eg. English ?
